lib/cuke_linter/linters/background_does_more_than_setup_linter.rb

```diff
@@ -8,7 +8,8 @@ class BackgroundDoesMoreThanSetupLinter < Linter
     def rule(model)
       return false unless model.is_a?(CukeModeler::Background)
 
-      model.steps.collect(&:keyword).any? { |keyword| (keyword == 'When') || (keyword == 'Then') }
+      dialect = DialectHelper.get_model_dialect(model)
+      model.steps.collect(&:keyword).any? { |keyword| (keyword == dialect.when_keyword) || (keyword == dialect.then_keyword) }
```

Comment:
The line might have to be something like
```
model.steps.collect(&:keyword).any? { |keyword| (dialect.when_keywords.include?(keyword)) || (dialect.then_keywords.include?(keyword)) }
```
in order to handle keyword synonyms but, yes, this is definitely the level of ease that I'd be shooting for when it comes to dealing with dialects. I'm just still up in the air as far as how to access the dialect object itself and where that will live. Should it be an attribute of all models (probably in `CukeModeler::Model`)? Should it be gotten through the `CukeModeler::Parsing` module? Somewhere else?

I suppose that if the dialect were inherently a part of the models then it wouldn't even be something configured in `cuke_linter`. It would just happen automatically when `cuke_modeler` generates models.